oshc/main/static/main/css/app.css

```diff
@@ -44,17 +49,31 @@ a {
   border-color: #24292e;
 }
 
+.container-fluid {
```

Comment:
Generally, it is a bad idea to apply styles to bootstrap classes. What if `container-fluid` is used somewhere else on the same page? It will automatically get `padding-left: 0px` which is not the intended behavior. You should add your own class/id if you wish to tweak a bootstrap element.